.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  release:
+    types: [published]
```

I prefer to do the build manually with a tag, without the needs to actually publish a release.
And also a manual trigger is always nice to have (probably we need some check, because Goreleaser will fail without a commit tag but still useful to trigger again a build.

```suggestion
on:
  workflow_dispatch:
  push:
    tags:
      - "v*"
```